### ISSUE TITLE
shiny demo

### ISSUE THREAD

**Issue by richfitz** [mwpennell/phyndr-ms#16]
In lieu of actually explaining the code, a shiny app that demos it with a simple tree and a table of Y/N for data so people can see what effect the algorithm would have.

**Comment by wcornwell** [comment]
@richfitz where do we start on this? sounds kinda fun coding project now that we sorta understand shiny

**Comment by richfitz** [comment]
First part is getting a good list of examples, rather that start with the shiny case, I think.

**Comment by wcornwell** [comment]
how many taxa?  10?

**Comment by richfitz** [comment]
Yeah, that's probably a good number - basically the size of the diagrams we were sketching when we started work on this.

**Comment by wcornwell** [comment]
from the land of schluter?

![image](https://cloud.githubusercontent.com/assets/5534161/9262922/221dd010-425c-11e5-9f03-1f8fed1e60dd.png)
